calico_node/startup.py

```diff
@@ -280,11 +284,12 @@ def main():
             ip = str(ips.pop())
         except IndexError:
             print "Couldn't autodetect a management IP address. Please " \
-                  "provide an IP address by rerunning the container with the" \
-                  " IP environment variable set."
+                  "provide an IP address either by configuring one in the " \
+                  "node resource, or by rerunning the container with the " \
+                  "IP environment variable set."
             sys.exit(1)
         else:
-            print "No IP provided. Using detected IP: %s" % ip
+            print "No IP configured or provided. Using detected IP: %s" % ip
```

Comment:
That is a valid fear - so I've added an "autodetect" option to the node run and to the container.